### PR TITLE
Update central_scheduler.rst

### PR DESCRIPTION
**Pull request by justinsousa**
## Description
We operate in a datacenter that was complaining about luigi running without authentication (called out in https://github.com/spotify/luigi/issues/2212) and listening for traffic on external interfaces (IP address). To get around this I passed `--address 127.0.0.1` to the luigi call in the systemd service file to allow the webserver to listen only on localhost. I noticed this flag wasn't documented so I figured I'd add it to this file so that others see it mentioned alongside the port flag. Someone else at my company noticed this issue cited above and didn't realize that the `--address` flag existed. 

## Motivation and Context
Basically, this is just to bring into the docs what I found by just running `luigi --help`. 

## Have you tested this? If so, how?
docs only update

### CODE REVIEW

**Review by dlstadther — APPROVED** [review]
Thanks for contributing back and making Luigi's documentation more complete!

**Comment by dlstadther** [comment]
Test failures are unrelated. Merging...
